includes/class-main.php

```
@@ -1553,8 +1553,15 @@ public function minify_css( $tag, $handle, $href ) {
 			$cached_file  = $css_minifier->minify();
 
 			if ( $cached_file ) {
+				static $content_url_base = array();
+				$blog_id                 = get_current_blog_id();
+
+				if ( ! isset( $content_url_base[ $blog_id ] ) ) {
+					$content_url_base[ $blog_id ] = content_url( 'cache/wppo/min/css/' );
+				}
+
 				$file_version = filemtime( Util::get_local_path( $cached_file ) );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'PerformanceOptimise\Inc\Util' in method 'minify_css'. (undefined)

(StaticAccess)